vars/assembleUpload.groovy

```diff
@@ -5,11 +5,7 @@ void call(Map args = [:]) {
     assembleManifest(args)
     uploadArtifacts(args)
 
-    withAWS(role: 'opensearch-bundle', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
-        s3Upload(bucket: "${ARTIFACT_BUCKET_NAME}", file: args.sha.lock, path: args.sha.path)
-    }
-
-    def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: args.sha.lock))
+    def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: args.manifest))
```

Comment:
If we are just uploading here can we change the file/lib name to `assemble.groovy`?

Reply:
This file stills `uploads` see line 6, so the name shouldn't change.  It just doesn't upload the lockfile anymore.

Reply:
This file stills `uploads` see line 6, so the name shouldn't change.  It no longer uploads lockfile.